src/Components/CookieConsent.js

```diff
@@ -0,0 +1,50 @@
+import * as React from "react";
+import * as Scrivito from "scrivito";
+import CookieConsent from "react-cookie-consent";
+
+/**
+ * @return {null}
+ */
+function CookieConsentBox() {
+  const root = Scrivito.Obj.root();
+  const cookieLink = root.get("cookiesPrivacyPolicyLink");
+  const cookiesLinkTitle = cookieLink && cookieLink.title();
+
+  if (!cookieLink) {
+    return null;
+  }
```

Comment:
You can also optimize the flow of code:

```js
  const cookieLink = root.get("cookiesPrivacyPolicyLink");
  if (!cookieLink) {
    return null;
  }

  const cookiesLinkTitle = cookieLink.title() || "Learn more »";
```

Later on, you would only use `cookiesLinkTitle` in the jsx part.